client-v2/src/shared/components/article/ArticleBodyDefault.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { styled } from 'shared/constants/theme';
 import startCase from 'lodash/startCase';
 import distanceInWordsStrict from 'date-fns/distance_in_words_strict';
+import { MdCollections } from 'react-icons/md';
```

Comment:
I believe a number of the icons we are using are from Material Design. Is is worth importing them all via react-icons like this rather than hosting our own svg files?  The library allows you to easily change color via context which is something I had to hack in a current PR #605 and it did not feel great.

Reply:
Yes, we should be consistent with how we import svgs. If everyone is happy with using this library I will make another changing this pr once this one is merged.